### DISCUSSION TITLE
Add the ability to provide release notes information when an update is available.

### DISCUSSION THREAD

**Discussion by marcocunha** [crazy-max/diun#1095] [Ideas]
Diun is working great and I got the notification via Slack. But they look a bit ugly and non  informative, like 

> @channel Docker tag [docker.io/library/mariadb:latest](https://github.com/MariaDB/mariadb-docker)  updated.
Hostname
c3bd483b5da9
Provider
docker
Created
Feb 11, 2024 23:03:42 UTC
Digest
sha256:a33fbcc7567325096c34d56aa085d69948f5bd0970244b855d4b027e38db083f
Platform
linux/arm64
HubLink
https://github.com/MariaDB/mariadb-docker
CrazyMax © 2024 Diun v4.26.0 | Today at 10:03

If we could add release notes information to this message, that would be great.

**Comment by spawn-guy** [comment]
I wonder how to do this...
Docker hub doesn't seem to have release notes, just versions.
There are links to github repo on dh, but not always present and not always to github.
And even then.. Not everyone adds "a description" or a release tag, but there are "commits" since last version.
